exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserServer.java

```diff
@@ -320,7 +348,7 @@ protected void consumeHandshake(ChannelHandlerContext ctx, UserToBitHandshake in
 
       @Override
       public BitToUserHandshake getHandshakeResponse(UserToBitHandshake inbound) throws Exception {
-        logger.trace("Handling handshake from user to bit. {}", inbound);
+        logger.trace("Handling handshake from user to bit. {}", safeLogString(inbound));
```

Comment:
Should we add `if (logger.isTraceEnabled()) {`? so `safeLogString` will be called only when we do need it for trace?